roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -167,6 +167,16 @@
 
 - include: update_master_config.yaml
 
+- name: "Copy Dashboards for Kibana"
+  copy:
+    src: roles/openshift_hosted_logging/files/dashboard.json
+    dest: ~/manageiq_ops_dashboard.json
+
+- name: "Push Dashboards to Kibana"
+  shell: "curl -X PUT -k -H \"kbn-version: 4.5.4\" -H \"Authorization: Bearer `{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig sa get-token management-admin -n management-infra`\" https://{{ openshift_hosted_logging_ops_hostname }}/elasticsearch/.kibana/dashboard/ManageIQOpsDashboard -d @manageiq_ops_dashboard.json"
```

Comment:
s/ManageIQOpsDashboard/ManageIQOpsOverview/ to avoid any indication this came from the "Department of the Redundancy Department".

Also, is this really only for ManageIQ?  Perhaps we should just name this the Ops Overview dashboard instead of including the ManageIQ name there, since it would be useful to have outside of ManageIQ?